xcodeproj/internal/xcodeproj_runner.bzl

```diff
@@ -15,7 +16,11 @@ def _xcodeproj_runner_impl(ctx):
         is_executable = True,
         substitutions = {
             "%bazel_path%": ctx.attr.bazel_path,
-            "%bazelrc%": ctx.file._bazelrc.path,
+            "%bazelrc%": ctx.file._bazelrc.short_path,
```

Comment:
Is `short_path` right for this one?

Reply:
I'll have to check. It was changed as part of something else that I reverted. It's probably wrong though :+1:.

Reply:
I've used `short_path` for all other substitutions, so I'm leaving this for now.